packages/core/saved-objects/core-saved-objects-import-export-server-internal/src/import/saved_objects_importer.ts

```diff
@@ -75,7 +84,19 @@ export class SavedObjectsImporter implements ISavedObjectsImporter {
       typeRegistry: this.#typeRegistry,
       importHooks: this.#importHooks,
       managed,
-    });
+    })
+      .then((response: SavedObjectsImportResponse) => {
+        this.#log.info(`🌸 Successfully imported ${response.successCount} saved objects.`);
```

Comment:
Move the logging into `importSavedObjectsFromStream` and  only log plain text:
```suggestion
        this.#log.info(`Successfully imported ${response.successCount} saved objects.`);
```